### DISCUSSION TITLE
Installation with defaults worked, after server reboot, can't connect to PostgreSQL

### DISCUSSION THREAD

**Discussion by ddorian** [pgsty/pigsty#432] [Q&A] [answered]
Hi team,

I did the simplest install possible on a single server in https://pigsty.io/docs/setup/install/.

I could login to the db with . But after I did a `sudo reboot` I can't login to the posgresql or pgbouncer with the same credentials.

Postgresql / patroni / etcd are all up though.

**Comment by waitingsong** [comment]
ip changed after reboot?

**Reply by ddorian** [reply]
The IP is static. I can't psql even from inside the vm using the same credentials. Maybe the hba conf changed?

```
# lsof -i :5432
COMMAND  PID     USER   FD   TYPE DEVICE SIZE/OFF NODE NAME
postgres 631 postgres    6u  IPv4  16777      0t0  TCP localhost:postgresql (LISTEN)
```

```
psql postgres://dbuser_dba:newpassword@localhost/meta
psql: error: connection to server at "localhost" (127.0.0.1), port 5432 failed: FATAL:  password authentication failed for user "dbuser_dba"
connection to server at "localhost" (127.0.0.1), port 5432 failed: FATAL:  password authentication failed for user "dbuser_dba"
```

```
psql postgres://dbuser_dba:new-password@static-ip/meta
psql: error: connection to server at "static-ip", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
```

**Comment by Vonng** [comment]
That's weird. what's your OS & pigsty version, and is that postgresql service launch by patroni or systemd?

**Reply by ddorian** [reply]
This is ubuntu 22.04, pigsty was just installed from https://pigsty.io/docs/setup/install/ .

> is that postgresql service launch by patroni or systemd?

The OS was clean. I guess it's managed by patroni/pigsty.

**Comment by Vonng** [comment, marked as answer]
Resloved in v3.0.0, the default systemd service is overwritten by the Deb package default.
